rollbar_dart/test/rollbar_test.dart

```diff
@@ -35,7 +30,7 @@ void main() {
         failingFunction();
       } catch (error, stackTrace) {
         await rollbar.error(error, stackTrace);
-        var payload = verify(await sender.send(captureAny)).captured.single;
+        var payload = verify(sender.send(captureAny)).captured.single;
```

Comment:
Why don't you want to `await` here? In other test cases, we do await.

```suggestion
        var payload = verify(await sender.send(captureAny)).captured.single;
```